tardis/montecarlo/tests/test_spectrum.py

```diff
@@ -166,44 +166,26 @@ def compare_spectra(actual, desired):
                 actual.distance,
                 desired.distance
                 )
-
-
-@pytest.mark.xfail
-def test_to_from_hdf(tmpdir, spectrum):
-    path = str(tmpdir.join('spectrum.hdf'))
-    spectrum.to_hdf(
-            str(path),
-            'spectrum'
-            )
-
-    spec_from = TARDISSpectrum.from_hdf(
-            path,
-            'spectrum'
-            )
-
-    compare_spectra(
-            spectrum,
-            spec_from)
-
-
-@pytest.mark.xfail
-def test_to_from_hdf_buffer(tmpdir, spectrum):
-    path = str(tmpdir.join('spectrum.hdf'))
-    spectrum.to_hdf(
-            str(path),
-            'spectrum'
-            )
-
-    with pd.HDFStore(path, mode='r') as buffer:
-        spec_from = TARDISSpectrum.from_hdf(
-                buffer,
-                'spectrum'
-                )
-
-    compare_spectra(
-            spectrum,
-            spec_from)
-
+                
+@pytest.fixture(scope="module")
+def hdf_file_path(tmpdir_factory):
+    path = tmpdir_factory.mktemp('hdf_buffer').join('spectrum.hdf')
+    return str(path)
+
+@pytest.fixture(autouse=True)
+def to_hdf_buffer(hdf_file_path,spectrum):
+    spectrum.to_hdf(hdf_file_path, name='spectrum')
+
+spectrum_attrs = ['_frequency', 'luminosity']
```

Comment:
I think we should use `TARDISSpectrum.hdf_properties` here. That way we don't have to update the tests if we change the `hdf_properties`.